contrib/update/update.yml

```diff
@@ -0,0 +1,110 @@
+---
+- name: create new instance
+  hosts: 127.0.0.1
```

Comment:
Add a hardcoded IP address is not good. Docker often changes the localhost's address, depending on the host system.
Proposed solution, either change it to localhost or have an inventory file[[1]](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html)(latest recommended).

Reply:
Hum, it does. As Docker Engine controls the emission of IPs, sometimes I get something slightly different than 127.0.0.1. Just remember that localhost actually is a CIDR of 8 (netmask 255.0.0.0).

Reply:
Any ideas on how to improve from `hosts: 127.0.0.1`? I mean, the Jarbas server has a fixed IP, could we use it instead, maybe set it in an envvar?

Reply:
Fixed by 1cd210110f5db236865775ae2c4714591ffaa811